Libraries/plist/Sources/Format/ASCIIPListLexer.cpp

```diff
@@ -216,7 +215,7 @@ ASCIIPListLexerReadData(ASCIIPListLexer *lexer)
 static int
 ASCIIPListLexerReadString(ASCIIPListLexer *lexer)
 {
-    if (lexer->pointer[0] == '\'') {
+    if (lexer->pointer[0] == '\'' && lexer->style == kASCIIPListLexerStyleASCII) {
         return ASCIIPListLexerReadSingleQuotedString(lexer);
     } else if (lexer->pointer[0] == '\"') {
```

Comment:
JSON keys are double-quoted:

    { "key" : "value" }

But not single-quoted:

     { 'key' : 'value' }

ASCII property lists support both.